client/src/components/ui/navigationbar/NavDrawer.svelte

```diff
@@ -0,0 +1,78 @@
+<script>
+    import NavQuery from './NavQuery.svelte';
+
+    export let show = false;
+    export let navItems;
+
+    let selected;
+</script>
+
+<nav class:show={!show}>
+  <div>
+        {#each navItems as item} 
+            <a href={`#/${item.href}`} on:click={() => {selected = item.href}} class={selected == item.href ? "selected" : ""}>
+                {item.label}
+            </a>
+      {/each}
+  </div>
+  <form method="POST" action="/auth/logout">
+      <input type="submit" value="Logout" />
+  </form>
+</nav>
+<style>
+    :global(body) {
+        padding: 0;
+    }
+
+    nav {
+        --acc-color: blue;
+        background-color: #ccc;
+        display: flex;
+        flex-direction: column;
+        justify-content: space-between;
+        position: absolute;
+        left: 0;
+        top: 75px;
+        z-index: 0;
+        transition: left 0.3s;
+        height: 100%;
+    }
+    nav.show {
+        left: -100%;
+    }
+  
+    a {
+        border-left: 0.5rem solid transparent;
+        color: initial;
+        display: block;
+        padding: 0.5rem;
+        width: 175pt;
+        text-decoration: none;
+        user-select: none;
+        transition: background-color 0.2s, border-left 0.2s;
+    }
+
+    a:hover {
+        background-color: #eee;
```

Comment:
Export as a color somewhere in `global.css` and then import as a `var(--some-color)`